File: JavaScript_Basic_04_2023_SolutionsOfProblems/03.Ex_PB-JS-Nested-Conditional-Statements/P03_NewHouse.js
function newHouse(input) {
    let flowersType = input[0];
    let flowersNumber = Number(input[1]);
    let budget = Number(input[2]);

    let flowersCost = 0.0;

    switch (flowersType) {
        case "Roses":
            flowersCost = flowersNumber * 5;
            if (flowersNumber > 80) {
                flowersCost = flowersCost * 0.9;
            }
            break;
        case "Dahlias":
            flowersCost = flowersNumber * 3.8;
            if (flowersNumber > 90) {
                flowersCost = flowersCost * 0.85;
            }
            break;
        case "Tulips":
            flowersCost = flowersNumber * 2.8;
            if (flowersNumber > 80) {
                flowersCost = flowersCost * 0.85;
            }
            break;
        case "Narcissus":
            flowersCost = flowersNumber * 3;
            if (flowersNumber < 120) {
                flowersCost = flowersCost * 1.15;
            }
            break;
        case "Gladiolus":
            flowersCost = flowersNumber * 2.5;
            if (flowersNumber < 80) {
                flowersCost = flowersCost * 1.2;
            }
            break;
    }
    if (flowersCost > budget) {
        console.log(`Not enough money, you need ${(flowersCost - budget).toFixed(2)} leva more.`);
    } else {
        console.log(`Hey, you have a great garden with ${flowersNumber} ${flowersType} and ${(budget - flowersCost).toFixed(2)} leva left.`);
    }
}

newHouse(["Narcissus",
"119",
"360"])

File: SoftUni_JAVA_BASIC_Jan_2023_SolutionsOfProblems/P03Ex_ConditionalStatements_Advanced/P03_NewHouse.java
package P03Ex_ConditionalStatements_Advanced;

import java.util.Scanner;

public class P03_NewHouse {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        String floursType = scanner.nextLine();
        int numberFlours = Integer.parseInt(scanner.nextLine());
        int budget = Integer.parseInt(scanner.nextLine());

        double amount = 0;
        switch (floursType) {
            case "Roses":
                amount = numberFlours * 5.0;
                if (numberFlours > 80) amount = amount * 0.9;
                break;
            case "Dahlias":
                amount = numberFlours * 3.80;
                if (numberFlours > 90) amount = amount * 0.85;
                break;
            case "Tulips":
                amount = numberFlours * 2.80;
                if (numberFlours > 80) amount = amount * 0.85;
                break;
            case "Narcissus":
                amount = numberFlours * 3.0;
                if (numberFlours < 120) amount = amount * 1.15;
                break;
            case "Gladiolus":
                amount = numberFlours * 2.5;
                if (numberFlours < 80) amount = amount * 1.2;
                break;
        }
        if (budget >= amount) {
            System.out.printf("Hey, you have a great garden with %d %s and %.2f leva left.",
                    numberFlours, floursType, budget - amount);
        } else {
            System.out.printf("Not enough money, you need %.2f leva more.", amount - budget);
        }
    }
}
